cookbook/bundles/dependencies.rst

```diff
@@ -0,0 +1,52 @@
+.. index::
+   single: Bundle; Dependencies
+
+How to Use Bundle Dependencies to load other Bundles
+====================================================
+
+.. versionadded:: 2.7
+    Bundle dependencies was introduced in Symfony 2.7.
```

Comment:
Then we should reword this to something like "Support for bundle dependencies was introduced [...]"